index.js

```diff
@@ -10,5 +14,18 @@ module.exports = async function httpRequestToUrl (request) {
   const proto = `http${socket.encrypted ? 's' : ''}:`
   const location = await socketLocation(socket)
 
+  if (isProxiedRequest(request) && opts.followProxies) {
+    return request.path
+  }
+
+  if (isProxiedRequest(request) && !opts.followProxies) {
+    return `${proto}//${location}`
```

Comment:
I'm unsure if this is _the right_ thing to do for all proxies when not returning the final destination.  Real world proxy servers can do some weird things.  But without real world data this feels like the right expedient thing.  I'm open to other suggestions here.

Reply:
No idea. Can always bugfix later, if something comes up. It's in the not-already-existing path anyway. 🤷